redoctober.go

```diff
@@ -765,6 +772,11 @@ var indexHtml = []byte(`<!DOCTYPE html>
 					data.Labels[i] = data.Labels[i].trim();
 					if (data.Labels[i] == "") { data.Labels.splice(i, 1); }
 				}
+				data.Usages = data.Usages.split(',');
+				for(var i=0, l=data.Usages.length; i<l; i++){
+					data.Usages[i] = data.Usages[i].trim();
+					if (data.Usages[i] == "") { data.Usages.splice(i, 1); }
```

Comment:
BTW, is there a workflow for updating this automatically from index.html? I read more about the rice tool, but it seems that's not used here.